src/sidebar/components/MarkdownEditor.js

```diff
@@ -267,7 +272,7 @@ function Toolbar({ isPreviewing, onCommand, onTogglePreview }) {
 
   return (
     <div
-      className="MarkdownEditor__toolbar"
+      className="hyp-u-layout-row hyp-u-padding--2 hyp-u-border--left hyp-u-border--right hyp-u-border--top hyp-u-bg-color--white MarkdownEditor__toolbar"
```

Comment:
The `hyp-u-padding` utility style ends up overriding the `MarkdownEditor__toolbar` styles' `pointer: coarse` media query because the utility styles are loaded after local components in the `sidebar.scss` entry point.

Master:

<img width="466" alt="Markdown toolbar buttons main" src="https://user-images.githubusercontent.com/2458/141795869-0b2c6528-580f-4e73-89ea-fdc7b2387d7a.png">

This branch:

<img width="443" alt="Markdown toolbar buttons branch" src="https://user-images.githubusercontent.com/2458/141795891-d86ee202-7dde-44d3-82de-43f09053bb4b.png">

Reply:
Great catch. Fixed.

Reply:
To clarify, this was fixed by removing the `hyp-u-padding*` styles from this component? The overall precedence of utility styles and component styles has not changed?

Reply:
Correct!